File: library/src/androidTest/java/com/raizlabs/android/dbflow/test/kotlin/KotlinDatabase.java
package com.raizlabs.android.dbflow.test.kotlin;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Description: Shows example for support for Kotlin
 */
@Database(version = KotlinDatabase.VERSION, name = KotlinDatabase.NAME,
        generatedClassSeparator = "_")
public class KotlinDatabase {

    public static final String NAME = "KotlinDatabase";

    public static final int VERSION = 1;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/kotlin/KotlinDatabase.kt
package com.raizlabs.android.dbflow.test.kotlin

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Description: Shows example for support for Kotlin
 */
@Database(version = KotlinDatabase.VERSION, name = KotlinDatabase.NAME,
    generatedClassSeparator = "_")
object KotlinDatabase {

    const val NAME = "KotlinDatabase"

    const val VERSION = 1
}
